### ISSUE TITLE
--smooth2d: continous and dashed lines are no longer aligned

### ISSUE THREAD

**Issue by mkarbach** [mkarbach/gammacombo#6]
When using the --smooth2d option, the continous and dashed lines are no longer aligned. The 2D plots are stacked plots, that have several instances of lines such that one always sees some part of the curve, even if laying behind another plot.

For some reason, smoothing affects both types differently, so that the dashed line becomes visible.

Reproduce with

    bin/tutorial --var a_gaus --var b_gaus -a pluginbatch --nrun 1 -c 4 --npoints 30 --npointstoy 30
    bin/tutorial --var a_gaus --var b_gaus -a plugin -j 1 -i -c 4 --smooth2d -a plot

![tutorial_tutorial4_a_gaus_b_gaus](https://cloud.githubusercontent.com/assets/4958479/5668782/8e6cee1c-9772-11e4-8643-12e598f7b40d.png)

**Comment by matthewkenzie** [comment]
Well spotted. I’ll have a look into it.

**Comment by matthewkenzie** [comment]
Should now be fixed by pull request #7. Running following lines:

``` bash
bin/tutorial --var a_gaus --var b_gaus -a pluginbatch --nrun 1 -c 4 --npoints 30 --npointstoy 30
bin/tutorial --var a_gaus --var b_gaus -a plugin -j 1 -i -c 4 --smooth2d -a plot
```

Will now produce:

![tutorial_tutorial4_a_gaus_b_gaus](https://cloud.githubusercontent.com/assets/1140576/5678153/ad756f6e-97ef-11e4-9d95-b10576fe9145.png)
